hello-spring/src/main/resources/openapi/api.yaml

```diff
@@ -4,7 +4,7 @@ info:
   description: A minimal "Hello, world" FOLIO module written in Java, inspired by the `hello-vertx` example.
   license:
     name: Apache 2.0
```

Comment:
Must also be same as POM: Apache License 2.0

Reply:
That one slipped by us – fixed!